com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -125,7 +125,7 @@ public bool IsCommunicatorOn
         List<ModelRunner> m_ModelRunners = new List<ModelRunner>();
 
         // Flag used to keep track of the first time the Academy is reset.
-        bool m_FirstAcademyReset;
+        bool HadFirstReset;
```

Comment:
nit: change back to m_FirstAcademyReset since that follows the naming convention

Reply:
Can I do m_HadFirstReset?

Reply:
Go for it